nmdc_server/static_files.py

```diff
@@ -0,0 +1,43 @@
+import importlib.resources
+import shutil
+from pathlib import Path
+
+from linkml_runtime import SchemaView
+from linkml_runtime.dumpers import json_dumper
+
+STATIC_PATH = Path("static")
+try:
+    shutil.rmtree(STATIC_PATH)
+except FileNotFoundError:
+    pass
+STATIC_PATH.mkdir(parents=True, exist_ok=True)
```

Comment:
Should this be wrapped in a function, and called explicitly when needed? That way we have more control over when we blow up the static files directory. Not an issue now but if this module ends up being imported elsewhere, it might cause headaches.

Reply:
Yeah good call. Let's not set ourselves up for foot-shooting in the future.